CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [6.0.0](https://github.com/puppeteer/puppeteer/compare/v5.5.0...v6.0.0) (2021-02-02)
+
+
+### ⚠ BREAKING CHANGES
+
+* **chromium:** The built-in `aria/` selector query handler doesn’t return ignored elements anymore.
```

Comment:
FYI @jackfranklin, re: our discussion from yesterday, in the future let's use [the suggested Conventional Commits format](https://www.conventionalcommits.org/en/v1.0.0/#summary) to mark breaking changes:

```
BREAKING CHANGE: Foo bar baz…
```

That way it automatically appears in the changelog, like this line right here. I’ll try to keep an eye out for this during reviews as well, this is all new for me too :)